storage/remote/azuread/azuread_test.go

```diff
@@ -124,8 +172,13 @@ func testGoodConfig(t *testing.T, filename string) {
 	}
 }
 
-func TestGoodAzureAdConfig(t *testing.T) {
-	filename := "testdata/azuread_good.yaml"
+func TestGoodAzureAdManagedIdentityConfig(t *testing.T) {
```

Comment:
Now that there's a bunch of these config tests, to make things more readable, do you mind putting all the cases in an array and then executing them like we do here (and a bunch of other places): https://github.com/prometheus/prometheus/blob/c5671c6d972ea961c6ba90be6f17171a661b5068/scrape/manager_test.go#L34-L444

Reply:
I updated the unit tests to follow the pattern you suggested

Reply:
Thanks! They're a lot easier to read now.

Reply:
Thanks!